frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -47,7 +47,7 @@
       <div class="columns is-centered pb-6">
         <div class="column is-three-fifths-desktop is-three-quarters-tablet is-fullwidth-mobile">
           <div class="content">
-            <h5 class="title is-5">You can search for a:</h5>
+            <h5 class="title is-5">With KEGG ids or EC codes, you can search for a:</h5>
```

Comment:
Is the `organism` value a KEGG id? For example on [this page](https://www.genome.jp/entry/gn:T03489), the value we have stored is `bmae` and the **Entry** listed on the linked page is actually `T03489`. It seems that the **Entry** field is the actual KEGG id, like on [this reaction page](https://www.genome.jp/entry/R03050).

Similarly for `domain`, it doesn't look like a KEGG id either.

Reply:
Thanks for the additional context @Yu-sysbio. My comment was mostly about the wording of the title `With KEGG ids or EC codes`. Just wondering if it should be changed since technically it's not just KEGG id and EC codes that we can search for.

Reply:
It is true that `KEGG id` (e.g. T03489) seems technically better than `Org code` (e.g. bmae) to search for an organism, but currently let us just not change since we are using `Org code` in our database. Normally, users would most likely search the full `Name` (e.g. Burkholderia mallei 6) for an organism, which we expect to be a future feature.

Reply:
Sorry that I was still unclear about this. I'm not suggesting to change anything in the data, I'm only talking about changing this line:

```html
<h5 class="title is-5">With KEGG ids or EC codes, you can search for a:</h5>
```

Reply:
There is an issue to improve the search, which would make this reference to KEGG ids obsolete, so I'm proposing to go back to the previous line:
```suggestion
            <h5 class="title is-5">One can search for a:</h5>
```